src/test/java/org/folio/rest/impl/protection/LinesProtectionTest.java

```diff
@@ -0,0 +1,86 @@
+package org.folio.rest.impl.protection;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static org.folio.orders.utils.ErrorCodes.ORDER_UNITS_NOT_FOUND;
+import static org.folio.orders.utils.ErrorCodes.USER_HAS_NO_PERMISSIONS;
+import static org.folio.rest.impl.PurchaseOrderLinesApiTest.LINES_PATH;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+import org.folio.HttpStatus;
+import org.folio.rest.jaxrs.model.Errors;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import io.restassured.http.Headers;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+
+@RunWith(JUnitParamsRunner.class)
+public class LinesProtectionTest extends ProtectedEntityTestBase {
+
+  private static final Logger logger = LoggerFactory.getLogger(LinesProtectionTest.class);
+
+  @Test
+  @Parameters({
+    "CREATE"
+  })
+  public void testOperationWithNonExistedUnits(ProtectedOperations operation) {
+    logger.info("=== Test corresponding order doesn't have units - expecting of call only to Assignments API ===");
```

Comment:
please correct the logger messages and the comments in this method

Reply:
done